scripts/macos-gen-dmg.py

```diff
@@ -26,6 +26,12 @@ def main(args):
         nargs='?',
     )
 
+    parser.add_argument('universal_dmg',
+        help='If specified, creates a universal dmg (x64/arm64)',
+        type=Path,
+        nargs='?',
+    )
```

Comment:
This argument is pretty misleading. What it actually does (or is meant to do) is use a pre-populated installation directory instead of making a new one. It should be called something like `installation`, with the help message updated accordingly.